### PR TITLE
Add WhatsApp.app version 0.1.0

### PR DESCRIPTION
**Pull request by vinkla**
WhatsApp client for Mac https://github.com/anatolinicolae/whatsapp

### CODE REVIEW

File: Casks/whatsapp.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'whatsapp' do
+  version '0.1.0'
+  sha256 '1c85878b726a84de33b04e2fa7139c71c6804a08a3b14bdd99b400d2f7bbea49'
+
+  url "https://github.com/anatolinicolae/whatsapp/releases/download/v#{version}/WhatsApp-v#{version}.zip"
+  name 'WhatsApp'
+  homepage 'https://github.com/anatolinicolae/whatsapp'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/whatsapp.rb]: `:mit`.

Comment (vinkla) [inline Casks/whatsapp.rb]: Done.

**Comment by ndr-qef** [comment]
Thanks, @vinkla, the cask is fine. However, I have reservations about the token: unofficial apps should not use trademarked names (WhatsApp) without prefix.

**Comment by vitorgalvao** [comment]
@ndr-qef I thought about that, but there is no official app for this. Using a trademarked name is an upstream problem (and one that should likely be discussed there). I’ll bring up again the notion that *we’re not a discoverability service, and you should use homebrew-cask to install packages that you’re aware of*.

**Pull request merged**